src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -745,7 +745,7 @@ export default class OrgPerson extends Mixins(CommonMixin, OrgPersonMixin) {
   /** Returns True if person name or org name has changed from its original properties. */
   protected hasNameChanged (orgPerson: OrgPersonIF): boolean {
     // is this a pre-existing person?
-    if (this.isPreExisting && this.isPerson) {
+    if (this.isFirmCorrectionFiling && this.isPreExisting && this.isPerson) {
```

Comment:
Please add another comment explaining that this check is for firm corrections only (does not apply to corps).

In which other filings is this component used and needs to be tested? Conversions maybe?

Reply:
What about the applicant for a limited restoration conversion?

Reply:
I will test other types of filings a bit more